modules/appeals_api/config/schemas/10182_headers.json

```diff
@@ -29,7 +29,7 @@
     "nodCreateHeadersSsn": {
       "allOf": [
         { "description": "social security number" },
-        { "type": "string", "pattern": "^[0-9]{9}$" }
+        { "type": "string", "maxLength": 9, "pattern": "^[0-9]{9}$" }
```

Comment:
Since the pattern expects an exact number of characters, let's enforce `minLength` too.